oddbird/oddcontrast
# Add color issues section

![](https://source.unsplash.com/featured/?cute,bird&burb)

## Description
Adds a section with space for more info on known color issues. Currently, only the Gamut mapping issue is documented- the Alpha caveats should be added with the related PRs.

Fixes #143 


## Steps to test/reproduce

1. Open the page on a wide screen, see that the section defaults to open.
2. Open the page on a narrow screen, see that the section defaults to closed.
3. Open/close is not responsive, so changing the viewport won't change the state
4. When the contrast doesn't meet AA Normal, it uses the same low contrast colors as the ratios.


## Show me
<img width="448" alt="image" src="https://github.com/oddbird/oddcontrast/assets/167908/a218318d-4c68-4fe9-883a-c7bd7124a190">
<img width="1035" alt="image" src="https://github.com/oddbird/oddcontrast/assets/167908/35161f48-fe3d-44c7-80b5-1a04df0504c5">
